spec/models/announcement_spec.rb

```diff
@@ -1,10 +1,11 @@
 require "rails_helper"
 
 RSpec.describe Refinery::Announcements::Announcement, :type => :model do
+  let(:today) { Time.zone.now }
 
   it "it has a tag with a title of 'news' ", js: true do
     tag1 = Refinery::Tags::Tag.create(title: 'news', narrative: "We do announcements.", tag_type: "content_type")
-    announcement1 = FactoryBot.create(:announcement, title: 'Test Announcement Title')
+    announcement1 = FactoryBot.create(:announcement, title: 'Test Announcement Title', published_date: today)
```

Comment:
Did you consider adding published_date to `vendor/extensions/announcements/spec/support/factories/refinery/announcements.rb` factory and then setting `published_date` to nil on the test for no date?

Reply:
Thanks! done.